lib/buffer.js

```diff
@@ -330,6 +334,52 @@ Buffer.from = function from(value, encodingOrOffset, length) {
   );
 };
 
+/**
+ * Creates the Buffer as a copy of the underlying ArrayBuffer of the view
+ * rather than the contents of the view.
+ * @param {ArrayBufferView} view
+ * @param {number} [offset]
+ * @param {number} [length]
+ * @returns {Buffer}
+ */
+Buffer.copyFrom = function copyFrom(view, offset, length) {
+  if (!isTypedArray(view)) {
+    throw new ERR_INVALID_ARG_TYPE('view', [ 'TypedArray' ], view);
+  }
+
+  const viewLength = TypedArrayPrototypeGetLength(view);
```

Comment:
Note that we _could_ defer on `TypedArrayPrototypeGetLength` to check wether `view` is a `TypedArray` or not as it would throw a `TypeError: Method get TypedArray.prototype.length called on incompatible receiver` anyway. But I get why we would prefer to have a Node.js specific error for consistency.